concepts/type-conversion/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Go allows to convert values from one type to another.",
```

Comment:
Slightly better grammar / simplification:

Go can convert values from one type to another.